public/js/main.js

```diff
@@ -57,13 +64,21 @@ $(document).on('ready', function() {
   }
 
   function updateFeaturedMembers() {
-
+    $.get('/data/featured_members.json', function(data) {
+      var members = data.members;
+      console.log(members[0]);
+      $('.member').each(function(index) {
+        var chosenMember = members[Math.floor(Math.random()*members.length)]; //Its the chosen one!
+        console.log(chosenMember);
+        members.splice(members.indexOf(chosenMember), 1);
+        $(this).find('h2').text(chosenMember.title);
```

Comment:
I'd like to get this PR deployed ASAP for Ben, is it okay if I keep this for the moment and work on adding the templating/middle where next?